Rakefile

```
@@ -1,17 +1,19 @@
 # frozen_string_literal: true
 
 require 'English'
-SWIFTLINT_VERSION = '0.27.0'
-XCODE_WORKSPACE = 'WordPress.xcworkspace'
-XCODE_SCHEME = 'WordPress'
-XCODE_CONFIGURATION = 'Debug'
-
 require 'fileutils'
 require 'tmpdir'
 require 'rake/clean'
 require 'yaml'
 require 'digest'
 
+RUBY_REPO_VERSION = File.read('./.ruby-version').strip
+SWIFTLINT_VERSION = '0.27.0'
+XCODE_WORKSPACE = 'WordPress.xcworkspace'
+XCODE_SCHEME = 'WordPress'
+XCODE_CONFIGURATION = 'Debug'
+EXPECTED_XCODE_VERSION = File.read('.xcversion')
```

Just to be on the safe side in case someone adds a CRLF at the end of this file in the future

```suggestion
EXPECTED_XCODE_VERSION = File.read('.xcversion').strip
```

---

[`922fedc` (#18845)](https://github.com/wordpress-mobile/WordPress-iOS/pull/18845/commits/922fedc092cfee3ce3f9d77a98ded5c0a19a6795)